src/tests/test_openapi.py

```diff
@@ -13,12 +13,22 @@ def test_root_view(api_client, afval_dataset, fietspaaltjes_dataset, filled_rout
     response = api_client.get(url)
     assert response.status_code == 200, response.data
 
-    assert response.data == {
-        "datasets": {
-            "afvalwegingen": "http://testserver/v1/afvalwegingen/",
-            "fietspaaltjes": "http://testserver/v1/fietspaaltjes/",
-        }
-    }
+    # Prove that response containes datasets
+    assert "datasets" in response.data
+
+    # Prove that datasets contains the datasets and noting else
+    assert set(response.data["datasets"].keys()) == set(["afvalwegingen","fietspaaltjes"])  
+
+    # Prove that both dataset elements are dictionaries
+    assert type(response.data["datasets"]["afvalwegingen"]) == dict
+    assert type(response.data["datasets"]["fietspaaltjes"]) == dict
```

Comment:
This breaks if and when we decide to use a different type, like a UserDict subclass. The fact that we can call keys below should be good enough.

Reply:
resolved in new commit